services/scan/src/store.ts

```diff
@@ -70,6 +72,7 @@ export enum ConfigKey {
   // @deprecated
   SkipElectionHashCheck = 'skipElectionHashCheck',
   PrecinctSelection = 'precinctSelection',
+  PollsState = 'pollsState',
```

Comment:
The polls state feels like a different class of data than the rest of the config values. Those are used to set up ("configure") the machine, while this is more a piece of operating state that has to do with using the machine (it's not a setting).

What do you think about storing it separately and using separate API routes? Does this seem too pedantic? The motivation would mostly just be clarity.

Reply:
It does seem too pedantic in isolation, but I'm thinking about how as part of this project I plan to move the last bits of frontend state (ballot replaced count and `isSoundsMuted`). Since we're going to have more of these values, I think it makes sense to add the clarity of breaking it out. 

What do you think about all settings being under a single API endpoint that you update in a `Partial` fashion? Just trying to limit code, but maybe that's the wrong motivation.

Reply:
Yeah, updating settings with a single endpoint seems fine.

Here's my take on how I would categorize the different bits of state:
- "Config" refers to setting up the machine to work in the right context (e.g. election definition, templates, precinct selection) (we might want to rename this to avoid confusion with machine config - machine id, etc)
- "Settings" are user settings that modify machine behavior, but don't have to do with the election (e.g. muting)
- "Operating state" tracks what's happening while the machine is being used (e.g. test mode, polls state) (this category maybe needs a better name)

Does that seem intuitive/clear to you, or would you think about it differently?

Reply:
This seems intuitive and clear to me, mostly. My only confusion is that, one way I've thought about config in the past is that it's something that affects how we interpret ballots. and "Test Mode" does affect how we read ballots. So on that piece, it's a big unclear. 

Given this is looking like a bigger change than I expected, I would prefer to do this in a different PR and merge the current polls open state as is, change later.

Reply:
FWIW I want to kill the `configs` table and replace it with specific tables/columns/etc for the data we want to store.

Reply:
@eventualbuddha where do you envision just one-off settings, like `isSoundMuted` living? All in their own column / table?

Reply:
Assuming we have an `elections` table similar to VxAdmin, these settings could either live on that table (if they're tied to an election) or in a `settings` table that's essentially a singleton table.

Reply:
I would prefer to separate out the reshuffling of how we store / access state from this PR. I propose I leave everything as-is for now, and I will revisit when I work on #2687, which is when I'll move the rest of the state back. Between now and then, we can sync for 15 min and sketch out how we'd like to lay out state?